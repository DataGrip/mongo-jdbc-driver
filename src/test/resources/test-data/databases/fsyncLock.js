// command
db.fsyncLock();
// command
db.fsyncUnlock();
