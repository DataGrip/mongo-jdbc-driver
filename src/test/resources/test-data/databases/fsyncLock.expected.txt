|map|
|Document{{info=now locked against writes, use db.fsyncUnlock() to unlock, lockCount=1, seeAlso=http://dochub.mongodb.org/core/fsynccommand, ok=1}}|

|map|
|Document{{info=fsyncUnlock completed, lockCount=0, ok=1}}|
